docs/docsite/rst/community/communication.rst

```diff
@@ -130,27 +83,24 @@ Working groups
 
 Working Groups are a way for Ansible community members to self-organize around particular topics of interest.
 
-Our community working groups are represented in `Matrix rooms <https://matrix.to/#/#space:ansible.com>`_ and  `Forum groups <https://forum.ansible.com/g>`_.
+Our community working groups are represented in `Forum groups <https://forum.ansible.com/g>`_ and additionally in `Matrix rooms <https://matrix.to/#/#space:ansible.com>`_.
```

Comment:
```suggestion
Our community working groups are represented in `Forum groups <https://forum.ansible.com/g>.
```

Reply:
The forum group pages should link to any relevant chat rooms.

Reply:
SGTM